Services/CO.CDP.EntityVerification.Tests/Services/PponServiceTests.cs

```diff
@@ -0,0 +1,19 @@
+using CO.CDP.EntityVerification.Services;
+
+namespace CO.CDP.EntityVerification.Tests.Services;
+
+public class PponServiceTests
+{
+    [Fact]
+    public void GeneratePponTest()
+    {
+        var scheme = "ppon";
+        var departmentIdentifier = "gcg";
+        PponService ppon = new PponService();
+        var idLength = 32;
+
+        string pponId = ppon.GeneratePponId(scheme, departmentIdentifier);
```

Comment:
Department identifier was optional in the spec, and we won't have it passed from supplier information (there's no concept of department in there). I'd just remove it until we need it. In the end, it's just appended to the generated identifier.

Reply:
Done